charts/vald/values.yaml

```diff
@@ -1019,7 +1019,10 @@ compressor:
     healths:
       liveness:
         enabled: false
-      readiness: {}
+      readiness:
+        server:
+          http:
+            shutdown_duration: 1m
```

Comment:
I think readiness should shutdown immediately. this setting increase duration until disconnect from kubernetes service dns.

Reply:
could you please describe more about why we need 1minutes duration for shutting down the readiness server?

Reply:
hmm. which field affects to make compressor pods alive longer without increasing duration until disconnect?

Reply:
Uh, I see. readiness should shutdown immediately so it should be 0s and for liveness, it affects pod to keep them alive.
but I know, we don't use liveness probe for agent and compressor, hmm.... 🤔

Reply:
ah ha, okay understood.
we have now a backup strategy in compressor post-stop, it may be okay to be terminated suddenly caused by unexpected fails of liveness servers. :thinking:

Reply:
I think so, maybe we need another phase for shutting down the process in internal server

Reply:
if we can set post pre process for each server, It would be useful

Reply:
i see. for now, i'm going to enable liveness and set readiness shutdown_duration to be zero. thanks

Reply:
sound good to me.
please set liveness shutdown duration over 1minutes.
and we need to think about pod disruption budget

Reply:
revised. please check it :smile: